### DISCUSSION TITLE
Display as cards, show more and be more intuitive.

### DISCUSSION THREAD

**Discussion by alex8088** [community/community#43921] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Display as cards, show more and be more intuitive.

**Comment by ahmadullah9983** [comment]
12121212 
 147258369 
 explorer 
 asdasdasd 
 aaaaaaaaaaaa 
 snickers 
 metallica 
 alexande 
 paradise 
 michigan 
 carolina 
 lacrosse 
 christin[]()https://www.facebook.com/profile.php?id=100070775316657&mibextid=ZbWKwL

**Comment by ahmadullah9983** [comment]
100070775316657afkhan12
100088919178585afkoko122

**Comment by Ogrodneybaker** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Display as cards, show more and be more intuitive.
>

**Comment by Satanas667** [comment]
Ygui

**Comment by pee563** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Display as cards, show more and be more intuitive.
> 
>

**Comment by Pramualsup22** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Display as cards, show more and be more intuitive.
>
